npm/ipsm/ipsm.go

```diff
@@ -70,6 +87,21 @@ func (ipsMgr *IpsetManager) Exists(listName string, setName string, kind string)
 	return true
 }
 
+// IpSetReferIncOrDec checks if an element exists in setMap/listMap and then increases or decreases tis refer count.
+func (ipsMgr *IpsetManager) IpSetReferIncOrDec(ipsetName string, kind string, countOperation ReferCountOperation) {
```

Comment:
It is better not to expose this function to other packages. IpSetReferIncOrDec() -> ipSetReferIncOrDec.

May Improve readability by making separate functions for increasing and decreasing reference count instead of using flag if possible, but, do not have strong opinion.

Reply:
We will need to expose this to network policy controller as netpol is the one referencing out ipsetmaps